inst/rstudio/templates/project/resources/common/index.Rmd

```diff
@@ -0,0 +1,55 @@
+--- 
+title: "A Minimal Book Example"
+author: "John Doe"
+date: "`r Sys.Date()`"
+site: bookdown::bookdown_site
+url: https://bookdown.org/johndoe/bookdown-demo
+cover-image: cover.png
+documentclass: book
+bibliography: [book.bib, packages.bib]
+yaml: goes here
```

Comment:
What's this?

Reply:
This is a placeholder mechanism so that we can insert the non common part of YAML header for index.Rmd between `gitbook()` and `bs4_book()`. 

My idea was to not duplicate so to have: 

* Common elements and format specific elements
* A mechanism to build the template from those two. 

So this placeholder is replaced in `bookdown_skeleton_insert_yml()` here: https://github.com/rstudio/bookdown/pull/1201/files#diff-f9db8366f31625a76f20758c5adc9d482d7a5243cb2ebe2d5cc2c9cd506fd7ccR61
I have done that to not add a dependency on a templating system like **whisker**

Maybe I should make that more clear by commenting or using a "non supported" YAML syntax ? 

```suggestion
<< placeholder >>
```

Do you have other ideas ? 

Or is aiming at not duplicate the common template project component a wrong idea ? 

The other mechanism is with the comment in Rmd files 
````
<!--bs4_book:start-->

Content specific to bs4_book()

<!--bs4_book:end-->
````

Reply:
Oh I see. I'm fine with `yaml: goes here`. I thought it was a YAML field that was accidentally left over.

Perhaps a comment like this will be more obvious:

```suggestion
# additional yaml goes here
```

Reply:
I'll make the change to a comment. That will be more explicit for future us and others

Reply:
@cderv can you resolve this when you make the change you have in mind?

Reply:
Change in 125b862